From b569653b6ccdecc587f8d3da3d538a6335ca34c7 Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Wed, 8 Apr 2015 23:19:00 +0200
Subject: [PATCH 045/310] fprobe-ulog:

mark as @BROKEN

error: missing ipt_ULOG.h header
ULOG was removed in Kernel since 3.17
fix in ulogd2 git:
https://git.netfilter.org/ulogd2/commit/?id=30e24dbfc7a8644e29664070e8c16e5c3997f87e

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 net/fprobe-ulog/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/fprobe-ulog/Makefile b/net/fprobe-ulog/Makefile
index 11afb5b..1ada611 100644
--- a/net/fprobe-ulog/Makefile
+++ b/net/fprobe-ulog/Makefile
@@ -20,7 +20,7 @@ include $(INCLUDE_DIR)/package.mk
 define Package/fprobe-ulog
   SECTION:=net
   CATEGORY:=Network
-  DEPENDS:=+libpthread
+  DEPENDS:=+libpthread @BROKEN
   TITLE:=NetFlow probe (ulog-based)
   URL:=http://fprobe.sourceforge.net
 endef
-- 
2.5.0

